### PR TITLE
用户的用户名和email 需要支持同时更新 

### PR DESCRIPTION
**Pull request by zzxr6**
用户的用户名和email 需要支持同时更新

### CODE REVIEW

File: app/javascript/components/popup/UpdateUsername.vue [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  const { cookies } = useCookies()
+  const { t } = useI18n()
+  const csghubServer = inject('csghubServer')
+  const nameRule = inject('nameRule')
+  const formData = ref({ username: '' })
+  const formRef = ref(null)
+
+  const getChangeUsername = computed(() => {
+    const canChangeUsername = cookies.get('can_change_username')
+    return canChangeUsername == 'true'
```
Comment (hiveer) [inline app/javascript/components/popup/UpdateUsername.vue:57]:
```suggestion
    return canChangeUsername === 'true'
```

File: app/javascript/components/popup/UpdateUsername.vue [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+            message: err.message,
+            type: 'warning'
+          })
+        })
+      }
+    })
+  }
+
+  const updateUsername = async () => {
+    const currentUsername = cookies.get('current_user')
```
Comment (hiveer) [inline app/javascript/components/popup/UpdateUsername.vue:120]: get current user from user Store

File: app/javascript/components/popup/UpdateUsername.vue (hunk 1 above)
Comment (hiveer) [reply app/javascript/components/popup/UpdateUsername.vue:57]: we should make sure the user is logged in, we can get this info from user Store

File: app/javascript/components/popup/UpdateUsername.vue [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+      </span>
+    </template>
+  </el-dialog>
+</template>
+
+<script setup>
+  import { ref, inject, computed } from 'vue'
+  import { useI18n } from 'vue-i18n'
+  import { useCookies } from 'vue3-cookies'
+  import jwtFetch from '../../packs/jwtFetch.js'
```
Comment (hiveer) [inline app/javascript/components/popup/UpdateUsername.vue:45]: use useFetchApi to replace jwtFetch

Comment (zzxr6) [reply app/javascript/components/popup/UpdateUsername.vue:45]: done

File: app/javascript/components/popup/UpdateUsername.vue (hunk 1 above)
Comment (zzxr6) [reply app/javascript/components/popup/UpdateUsername.vue:57]: done

File: app/javascript/components/popup/UpdateUsername.vue (hunk 2 above)
Comment (zzxr6) [reply app/javascript/components/popup/UpdateUsername.vue]: done

**Review by hiveer — APPROVED** [review]
LGTM

**Pull request merged**